actions/add_book.rb

```diff
@@ -0,0 +1,27 @@
+require './classes/book'
+require './inputs/item_inputs'
+require './inputs/book_inputs'
+require './actions/instance_creator'
+require 'json'
+
+class AddBook
+  def self.add_book(books)
+    item_name, item_genre, item_author_first, item_author_last, item_label_title,
+    item_label_color, item_date = ItemInputs.item_inputs
+    book_publisher, book_cover_state = BookInputs.book_inputs
+
+    book_instance = Book.new(name: item_name, publisher: book_publisher, cover_state: book_cover_state,
+                             genre: {}, author: {}, label: {}, publish_date: item_date)
+
+    genre = InstanceCreator.genre_instance(item_genre, book_instance)
+    author = InstanceCreator.author_instance(item_author_first, item_author_last, book_instance)
+    label = InstanceCreator.label_instance(item_label_title, item_label_color, book_instance)
+
+    book = { 'json_class' => 'book', 'title' => book_instance.name, 'genre' => genre,
+             'author' => author, 'Cover State' => book_instance.cover_state,
+             'label' => label, 'publisher' => book_instance.publisher, 'publish date' => book_instance.publish_date,
+             'id' => book_instance.id, 'archived' => book_instance.archive }
+    books << book
+    puts 'Book Created'
+  end
+end
```

Comment:
- [ ] I deleted all the .json files and I tried creating a book, music, and game but the app crashed. Please let's make sure that the app does not crash if the .json file does not exist or is empty.

<img width="301" alt="Loom_C5VyLTbLB2" src="https://user-images.githubusercontent.com/81468675/153517011-e32535eb-6510-43b4-a18d-e9d6cc8ef02c.png">

- [ ] I received this error below too when I tried running the app with no .json files. Please let's fix these errors 💪
<img width="287" alt="Loom_B9ykdzqV8q" src="https://user-images.githubusercontent.com/81468675/153517307-07799b63-47e3-46e8-960d-e97ba3882d4c.png">

Reply:
thanks for your review, we'll work to fix it!